Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -62,14 +64,16 @@ const Info<ExpansionInterface::EXIDeviceType> MAIN_SLOT_B{{System::Main, "Core",
 const Info<ExpansionInterface::EXIDeviceType> MAIN_SERIAL_PORT_1{
     {System::Main, "Core", "SerialPort1"}, ExpansionInterface::EXIDeviceType::None};
 
-const Info<ExpansionInterface::EXIDeviceType>& GetInfoForEXIDevice(int channel)
+const Info<ExpansionInterface::EXIDeviceType>& GetInfoForEXIDevice(ExpansionInterface::Slot slot)
 {
-  static constexpr std::array<const Info<ExpansionInterface::EXIDeviceType>*, 3> infos{
-      &MAIN_SLOT_A,
-      &MAIN_SLOT_B,
-      &MAIN_SERIAL_PORT_1,
-  };
-  return *infos[channel];
+  static constexpr Common::EnumMap<const Info<ExpansionInterface::EXIDeviceType>*,
+                                   ExpansionInterface::MAX_SLOT>
```

Comment:
This isn't right, `ExpansionInterface::MAX_SLOT` is currently numerically `2`. I assume it should be `3`?

Reply:
Ooh wait, nevermind, I didn't parse that this is an `EnumMap` now instead of an `array`.